lib/components/SLocalNav.vue

```diff
@@ -0,0 +1,39 @@
+<script setup lang="ts">
+import SLocalNavMenu, { type MenuItem } from './SLocalNavMenu.vue'
+import SLocalNavTitle, { type Title } from './SLocalNavTitle.vue'
+
+export type { Title, MenuItem }
+
+defineProps<{
+  title: Title[]
+  menu?: MenuItem[][]
+}>()
+</script>
+
+<template>
+  <div class="SLocalNav" :class="{ 'has-menu': menu }">
+    <SLocalNavTitle :title="title" />
+    <SLocalNavMenu v-if="menu" :menu="menu" />
+  </div>
+</template>
```

Comment:
You might want to add `has-menu` class and render `SLocalNavMenu` based on length of `menu` instead of `menu` existence, because `menu` array can be an empty under a certain condition and render empty `SLocalNavMenu` with 0 padding bottom, which I believe you don't want.

Reply:
Good point. Fixed 👍